src/layers/normalise.jl

```diff
@@ -34,7 +34,7 @@ The [`Dropout`](@ref) layer is what you should use in most scenarios.
 function dropout(rng, x, p; dims=:, active::Bool=true)
   active || return x
   y = dropout_mask(rng, x, p, dims=dims)
-  return x .* y
+  return x .* iszero.(y)
```

Comment:
```suggestion
  return x .* y
```
This change is incorrect, since `y` is not a binary mask (it contains scaling factors to compensate for the dropped elements).